### PR TITLE
Enhancement of zh_HK.langcfg

### PR DESCRIPTION
**Pull request by Lunari8546**

### CODE REVIEW

**Comment by HTGAzureX1212** [comment]
I am not very sure about the "Server Settings" translation for "Guild Features" 🤔

**Comment by Lunari8546** [comment]
Me either, any ideas?

**Comment by HTGAzureX1212** [comment]
"Guild Features" are features that Discord guilds can optionally enable; for example Community and Threads. Not sure how would we express that in the translations.

**Comment by Lunari8546** [comment]
`Server 設定` seems good to me tho sounds weird comparing to the original.

**Comment by HTGAzureX1212** [comment]
> `Server 設定` seems good to me tho sounds weird comparing to the original.

Especially when taking into consideration the actual meaning, those translations seem weird.

File: langcfgs/zh_HK.langcfg [hunk 1]
```diff
@@ -55,7 +55,7 @@ InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldName: Server 嘅 chann
 InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueCategoriesPart: 類別
 InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueTextChannelsPart: 文字 Channels
 InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueVoiceChannelsPart: 語音 Channels
-InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueStageChannelsPart: 舞台
+InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueStageChannelsPart: 舞台 Channels
 InformationPlugin.GuildinfoCommand.EmbedGuildChannelsFieldValueNewsChannelsPart: 通知/新聞 Channels
-InformationPlugin.GuildinfoCommand.EmbedGuildFeaturesFieldName: Server 特點
-InformationPlugin.GuildinfoCommand.EmbedGuildVerificationLevelFieldName: Server 驗證 Lv
+InformationPlugin.GuildinfoCommand.EmbedGuildFeaturesFieldName: Server 設置
```
Comment (HTGAzureX1212) [inline langcfgs/zh_HK.langcfg]:
```suggestion
InformationPlugin.GuildinfoCommand.EmbedGuildFeaturesFieldName: Server 開左嘅附加 Discord 功能
```

Does this translation make sense?

Comment (Lunari8546) [reply langcfgs/zh_HK.langcfg]: Yes, but mayhaps `Server 嘅附加 Discord 功能` would be better.

Comment (HTGAzureX1212) [reply langcfgs/zh_HK.langcfg]: Makes sense. Well, let's go for that translation then.

**Review by HTGAzureX1212 — APPROVED** [review]

**Pull request merged**